harness/determined/cli/trial.py

```diff
@@ -89,10 +83,15 @@ def _workloads_tabulate(
             if metrics_workload:
                 row_metrics = [json.dumps(metrics_workload.metrics.to_json(), indent=4)]
 
+        if isinstance(w_unpacked, bindings.v1CheckpointWorkload):
+            state = _format_state(w_unpacked.state)
+        else:
+            state = "COMPLETED"
+
         values.append(
             [
                 w_unpacked.totalBatches,
-                _format_state(w_unpacked.state),
+                state,
```

Comment:
can we just remove this entirely, here and elsewhere where it is rendered/checked?